### PR TITLE
Input validation

### LINKED ISSUE #81: Gui bugs/missing features
- [x] Movement margin is missing a unit identifier (seconds/counts)
- [x] ScriptingProject is missing a name
- [x] CameraTimeLine is missing a name
- [x] Add descriptions to cameratypes and cameratimelines tables in new project
- [x] Empty strings give exception, please validate this and give feedback to the user

### PR DESCRIPTION
**Pull request by mennoo1996**
Fixes #81 
Added names to ScriptingProject and CameraTimeline
Added unit to movementmargin in modal
Description of cameratype, camera and timeline are now shown in the create project modal
Input is validated when creating a new project

### CODE REVIEW

File: src/main/java/control/FileMenuController.java [hunk 1]
```diff
@@ -219,39 +285,114 @@ private void addTimeline(MouseEvent event) {
...
+        timelineModal.getErrorLabel().setTextFill(Color.RED);
+        
+        return errorString.isEmpty();
     }
     
     /**
      * Event handler for when the create project button is clicked in the NewProjectModalView.
      * @param event the mouse event related to this event
      */
     private void createProject(MouseEvent event) {
```
Comment (bart-de-jonge) [inline src/main/java/control/FileMenuController.java:336]: This method is quite long but has no whitespaces and/or comments, could you add a few for readability

File: src/main/java/gui/modal/AddCameraTypeModalView.java [hunk 2]
```diff
@@ -30,6 +30,9 @@
     @Getter
     private VBox viewPane;
     
+    @Getter
+    private Label errorLabel;
+
```
Comment (bart-de-jonge) [inline src/main/java/gui/modal/AddCameraTypeModalView.java]: 2 empty lines

File: src/main/java/gui/modal/NewProjectModalView.java [hunk 3]
```diff
@@ -68,6 +71,9 @@
     @Getter
     private ArrayList<CameraTimeline> timelines;
     
+    @Getter
+    private Label errorLabel;
+
```
Comment (bart-de-jonge) [inline src/main/java/gui/modal/NewProjectModalView.java:76]: 2 empty lines

**Comment by bart-de-jonge** [comment]
A few minor comments, but other than that great work

**Comment by martijnjanssen** [comment]
Looks good.

**Pull request merged**